nixos/modules/services/monitoring/gatus.nix

```diff
@@ -0,0 +1,127 @@
+{ pkgs, lib, config, ... }:
```

Comment:
Could you please pass this new file through `nixfmt-rfc-style` ?

Reply:
Done!